README.md

```diff
@@ -97,7 +97,7 @@ once a problem is found by a reviewer, it goes through the following stages:
 * **reviewer** creates github issue in some repository, at their discression
 * if necessary, **maintainer** moves the issue to a more suitable repository
 * **maintainer** assigns the milestone, if necessary after consulatation with **reviewer** and other **stakeholders**  and **reviewers**
-* if issue is in the current milestone, **developer** fixes it at their convenience and assigns the issue to the **reviewer**
+* if issue is in the current milestone, **developer** fixes it at their convenience and asks the **reviewer** for a review of the applied changes 
```

Comment:
I mean preciesly `assigning` as in the github issue mechanism. It may be clarified that this `assigning` action means asking, but I think it should say explicitly `assigning` to explain how to proceed.

Reply:
Then maybe I got confused with the review process of a PR. 

Is that what you imply with "**developer** fixes it"? Or, for the current context, it is irrelevant how this fix/change is applied?

Reply:
No, really, it is **developer** that fixes it, but then redirects (`assigns`) the issue to the **reviewer** who should make the further step. As discussed today.